heap/max_heap.py

```diff
@@ -34,16 +34,15 @@ def _sift_down(self, index):
         lc = (index * 2) + 1
         rc = (index * 2) + 2
         if lc < len(store):
-            if rc < len(store):
-                if store[rc] > store[lc]:
-                    if store[rc] > store[index]:
-                        store[index], store[rc] = store[rc], store[index]
-                        return self._sift_down(rc)
-                    else:
-                        return
-                if store[lc] > store[rc]:
-                    if store[lc] > store[index]:
-                        store[index], store[lc] = store[lc], store[index]
-                        return self._sift_down(lc)
-                    else:
-                        return
+            if rc < len(store) and store[rc] > store[lc]:
```

Comment:
why are you sifting down on `get_max()`, if you're already sifting down on line 13?
I may be not reading this well, but it seems like you're sifting down more than you need to.